pkg/apis/kai/v1/config_types.go

```diff
@@ -24,19 +24,22 @@ import (
 type ConditionType string
 
 const (
-	ConditionTypeReconciling       ConditionType = "Reconciling"
-	ConditionTypeDeployed          ConditionType = "Deployed"
-	ConditionTypeAvailable         ConditionType = "Available"
-	ConditionDependenciesFulfilled ConditionType = "DependenciesFulfilled"
+	ConditionTypeReconciling            ConditionType = "Reconciling"
+	ConditionTypeDeployed               ConditionType = "Deployed"
+	ConditionTypeAvailable              ConditionType = "Available"
+	ConditionDependenciesFulfilled      ConditionType = "DependenciesFulfilled"
+	ConditionTypePrometheusConnectivity ConditionType = "PrometheusConnectivity"
```

Comment:
This shouldn't be be a condition, but part of the message. When Prometheus can't connect we should be "unavailable"